service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/AmendedOrderEventHandlerTest.java

```diff
@@ -97,40 +97,53 @@ void before() {
 
     @Test
     @SuppressWarnings("unchecked")
-    void shouldNotifyPartiesOnOrderSubmission() {
-        given(representativesInbox.getEmailsByPreference(CASE_DATA, EMAIL)).willReturn(EMAIL_REPS);
-        given(representativesInbox.getEmailsByPreference(CASE_DATA, DIGITAL_SERVICE)).willReturn(DIGITAL_REPS);
-        given(otherRecipientsInbox.getNonSelectedRecipients(eq(EMAIL),
-            eq(CASE_DATA),
-            eq(SELECTED_OTHERS),
-            any()))
-            .willReturn((Set) Set.of(EMAIL_REP_1));
-        given(otherRecipientsInbox.getNonSelectedRecipients(eq(DIGITAL_SERVICE),
-            eq(CASE_DATA),
-            eq(SELECTED_OTHERS),
-            any()))
-            .willReturn((Set) Set.of(DIGITAL_REP_1));
-
-        underTest.notifyParties(EVENT);
+    void shouldNotifyLocalAuthorityWhenOrderAmended() {
+        underTest.notifyLocalAuthority(EVENT);
 
         verify(notificationService).sendEmail(
             ORDER_AMENDED_NOTIFICATION_TEMPLATE,
             Set.of(LOCAL_AUTHORITY_EMAIL_ADDRESS),
             NOTIFY_DATA,
             CASE_ID.toString()
         );
+    }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    void shouldNotifyEmailRepsWhenOrderAmended() {
+        given(representativesInbox.getEmailsByPreference(CASE_DATA, EMAIL)).willReturn(EMAIL_REPS);
+        given(otherRecipientsInbox.getNonSelectedRecipients(eq(EMAIL),
+            eq(CASE_DATA),
+            eq(SELECTED_OTHERS),
+            any()))
+            .willReturn((Set) Set.of(EMAIL_REP_1));
+
+        underTest.notifyEmailRepresentatives(EVENT);
 
         verify(representativeNotificationService).sendNotificationToRepresentatives(
             CASE_ID,
             NOTIFY_DATA,
-            Set.of(DIGITAL_REP_2),
+            Set.of(EMAIL_REP_2),
             ORDER_AMENDED_NOTIFICATION_TEMPLATE
         );
+    }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    void shouldNotifyDigitalRepsWhenOrderAmended() {
+        given(representativesInbox.getEmailsByPreference(CASE_DATA, DIGITAL_SERVICE)).willReturn(DIGITAL_REPS);
+        given(otherRecipientsInbox.getNonSelectedRecipients(eq(DIGITAL_SERVICE),
+            eq(CASE_DATA),
+            eq(SELECTED_OTHERS),
+            any()))
+            .willReturn((Set) Set.of(DIGITAL_REP_1));
```

Comment:
might want to suppress raw class usage here if it is present, same for other places in this file